compute/instances/how-to/migrate-routed-ips.mdx

```diff
@@ -0,0 +1,50 @@
+---
+meta:
+  title: How to migrate an Instance to routed flexible IPs
+  description: This page explains how to migrate your Instance to routed flexible IPs
+content:
+  h1: How to migrate an Instance to routed flexible IPs
+  paragraph: This page explains how to migrate your Instance to routed flexible IPs
+tags: instance flexible ip routed
+dates:
+  validation: 2023-10-11
+  posted: 2023-10-11
+categories:
+  - compute
+---
+
+During deployment, each Instance gets a private IP from its hosting node, besides its public IP that is routed to the Instance through a carrier-grate NAT solution. However, if the underlying hypervisor changes, the private IP changes. This can happen when you restart your Instance. The private IP lets Instances communicate via the underlying private network, without public IPs. 
+
+Routing a public IP directly to an Instance bypasses private IPs, sending traffic to its routed flexible IP. The benefit is the IP stays consistent even if the Instance is restarted or moved. Routed IPs also let you assign multiple flexible IPv4s and IPv6s to an Instance, as both IPv4 and IPv6 are supported.
+
```

Comment:
```suggestion


Routed IPs are the preferred choice over NATed IPs because they offer numerous advantages, including simplified network configuration, improved performance, direct device accessibility, and enhanced end-to-end communication transparency.

```

@RoRoJ ok for you?